tmt/steps/report/__init__.py

```diff
@@ -95,7 +103,7 @@ def base_command(cls, method_class=None, usage=None):
         @click.option(
             '-h', '--how', metavar='METHOD',
             help='Use specified method for results reporting.')
-        def report(context, **kwargs):
+        def report(context: Any, **kwargs: Any) -> None:
```

Comment:
@happz, is there a type for `click`'s context object? Or are we good with `Any` here?

Reply:
Yep, `click.Context` :)

Every single `Any` needs to be examined, it's like an admission of defeat, and we shall not give up so easily :)

Reply:
Updated, thanks!